sagemathinc/cocalc
# Add persistence to user search in admin page

# Description
Add persistence to just the user search.

Does not include persistence for extra drop downs from the user search.

# Testing Steps
1. Open the admin panel
1. Search for a person
1. Leave the admen page and come back.
1. The search should still be there.

# Relevant Issues

### [Checklist](https://github.com/sagemathinc/cocalc/wiki/PR-Checklist):

Front end:
